### DISCUSSION TITLE
Special Characters in blade.php

### DISCUSSION THREAD

**Discussion by MrBaker10** [ismaelw/laratex#10] [Q&A]
` 
\documentclass[a4paper,9pt,landscape]{article}
\usepackage{adjustbox}
\usepackage[german]{babel}
\usepackage[scaled=.92]{helvet}
\usepackage{fancyhdr}
\usepackage[svgnames,table]{xcolor}
\usepackage[a4paper,inner=1.5cm,outer=1.5cm,top=1cm,bottom=1cm,bindingoffset=0cm]{geometry}
\geometry{textwidth=\paperwidth, textheight=\paperheight, noheadfoot, nomarginpar}

\renewcommand{\familydefault}{\sfdefault}

\pagestyle{fancy}
\fancyhead{}
\renewcommand{\headrulewidth}{0pt}
\fancyfoot{}
\fancyfoot[LE,RO]{\thepage}

\fancyfoot[C]{\fontsize{8pt}{8pt}\selectfont The above document is auto-generated.}
\renewcommand{\footrulewidth}{0.2pt}

\begin{document}

    \section*{\centering{LaraTeX Demo Document}}
    
    \begin{center}
        \item[Actual year:] {{ $year }}
    \end{center}
    
    
    \begin{table}[ht]
        \centering
        \renewcommand{\arraystretch}{2}
        \begin{tabular}{|c|l|l|l|l|l|l|l|l|l|l|} 
             \hline
             \rowcolor[HTML]{E3E3E3}
             LFDNR & Finanzposition & Buchung & Jahr & Empfänger & Bg-Nummer & Nachname & Vorname & Betrag & Plz & Wohnort \\
             \hline\renewcommand{\arraystretch}{1.5}
             
          @foreach ($items as $item) 
            {{ $item->id}} &
            {{ $item->Finanzposition}} &
            {{ $item->Buchung->format('d.m.Y')}} &
            {{ $item->HJ}} &
            {{ $item->Empfänger}} & 
            {{ $item->BG_Nummer}} &
            {{ $item->Nachname}} &
            {{ $item->Vorname}} &
            {{ $item->Betrag}} € &
            {{ $item->PLZ}} &
            {{ $item->Wohnort}} \\ \hline
        @endforeach
    
        \end{tabular}
        \caption{\textbf{Die Gesamtsumme beträgt {{$sum}}€}.}
    \end{table}
    

\end{document}
`

This is my tex.blade.php

Special characters occur in the data in the foreach loop.
This results in errors like "Missing $ inserted" and "You can't use `macro parameter character #' in restricted horizontal mode".

The special characters are " ' " and " _ ". How do I allow the package to simply convert these characters from the foreach loop into the PDF.

**Comment by ismaelw** [comment]
It would be interesting to see the rendered file. It might be that you need to wrap certain special characters.
Can you try to disable the teardown setting (set it to false in config) and show me the rendered file. I am not a LaTeX specialist and to see the rendered final tex file could help troubleshooting this.

**Comment by MrBaker10** [comment]
My problem is, I don't know where the character "#" comes from. I can't find anything about it when debugging

**Reply by ismaelw** [reply]
Hi @MrBaker10 
To figure out where this character comes from you can anyway disable teardown so on your FTP Server, in the temp Folder you defined in the config, it generates the final TEX file. This TEX file will show you, where the special Character is located (This tex file is generated before converting TEX to PDF, so before you receive your TEX-Exception).

**Comment by ismaelw** [comment]
Hi @MrBaker10 feel free to reply to my suggestion. Otherwise I will close this discussion soon due to inactivity.
